cmd/prom2hny/main.go

```diff
@@ -88,6 +85,43 @@ func NewMetricGroups(mfs []*dto.MetricFamily) []*MetricGroup {
 	return metricGroups
 }
 
+// We can't use m.GetGauge().GetValue() because there exist some
+// exceptional cases we need to handle. Abstract exception cases
+// in this func
+func getDatapointFromMetric(mf *dto.MetricFamily, m *dto.Metric) *DataPoint {
+	metricLabels := makeLabels(m)
+	metricName := mf.GetName()
+	var metricValue interface{}
+
+	switch metricName {
+	case "kube_pod_status_phase":
+		metricValue = metricLabels["phase"]
+		delete(metricLabels, "phase")
+
+	// Only contribute labels
+	case "kube_pod_labels", "kube_pod_info", "kube_service_info", "kube_pod_container_info", "kube_node_labels":
+		metricValue = nil
+
+	// Formatted as Condition Values
+	case "kube_pod_status_ready", "kube_pod_status_scheduled", "kube_node_status_disk_pressure", "kube_node_status_memory_pressure", "kube_node_status_out_of_disk", "kube_node_status_ready":
+		if m.GetGauge().GetValue() == 1 {
+			metricValue = metricLabels["condition"]
```

Comment:
We could. The interface for "condition" metrics isn't documented, so a condition could be a non-bool value one day. That probably won't happen, but who knows.

One of the reasons for not doing this is because I've reserved nil as a value for not including the metric (only it's labels). https://github.com/honeycombio/prom2hny/pull/3/files#diff-9042887df5c691cf03b2dd8a5e92912fR180

I don't have strong feelings about it, but if you do, I can change it such that "true" and "false" are represented as bools